docs/react-native/docs/triggers.md

```diff
@@ -143,16 +143,32 @@ const trigger: TimestampTrigger = {
   //...
   alarmManager: {
     allowWhileIdle: true,
-  }
+  },
 };
 ```
 
 Please note, for iOS, a repeating trigger does not work the same as Android - the initial trigger cannot be delayed:
+
 - `HOURLY`: the starting date and hour will be ignored, and only the minutes and seconds will be taken into the account. If the timestamp is set to trigger in 3 hours and repeat every 5th minute of the hour, the alert will not fire in 3 hours, but will instead fire immediately on the next 5th minute of the hour.
 - `DAILY`: the starting day will be ignored, and only the time will be taken into account. If it is January 1 at 10 AM and you schedule a daily recurring notification for January 2 at 11 AM, it will fire on January 1 at 11 AM and every day thereafter.
 - `WEEKLY`: the starting week will be ignored, and only the day and time will be taken into account.
 
->  For more details, please see the discussion [here](https://github.com/notifee/react-native-notifee/issues/241).
+> For more details, please see the discussion [here](https://github.com/notifee/react-native-notifee/issues/241).
+
+### Android 12 Limitations
+
+Starting from Android 12, timestamp triggers cannot be created unless user specfically allow the [exact alarm permission](https://developer.android.com/about/versions/12/behavior-changes-12).
```

Comment:
I don't feel really strongly here, but I slightly prefer this link - it contains a link to more info which is the original link you had here, but this is the stable documentation reference in case the behavior changes docs go away or move for whatever reason?

```suggestion
Starting from Android 12, timestamp triggers cannot be created unless user specfically allow the [exact alarm permission](https://developer.android.com/reference/android/Manifest.permission#SCHEDULE_EXACT_ALARM).
```

Reply:
That's great and the new link also has reference to behavior changes in itself, so it should suffice